static/css/marble.css

```diff
@@ -964,6 +943,16 @@ h6{
 
 /*Media Queries*/
 @media (width < 576px){
+    /*Body Components*/
+    .header-title-margin{
```

Comment:
This is identical in all widths, move this out of the media widths.

Reply:
This is 50px at >992px media query.